tests/acceptance/28_inform_testing/common.cf.sub

```diff
@@ -0,0 +1,77 @@
+body common control
+{
+  # seems weird to have ../../ here, but it is relative to the test that includes this file
+  # which will "always" be a direct sub-dir of this current dir
+  inputs => { "../../default.cf.sub" };
```

Comment:
Why not use `$(this.promise_dirname)`?

Reply:
I'm pretty sure it was reflecting the test.cf promise_dirname, but I will try again to be sure.

Reply:
`$(this.promise_dirname)` isn't available in `body file control` or `body common control` so have to do two-chunk style work to resolve the var and then use as inputs. I think I have addressed your concern about using paths relative to a var and not just relative to "whatever", right?